.github/markets/pr_template.md

```diff
@@ -66,3 +66,10 @@ Each json file under the [configs](../../configs) folder correspond to their res
 |`baseDenom`   |`string`  |true   |The denom of this token on its native chain (e.g. `uosmo` for $OSMO on Osmosis, `uatom` for $ATOM on CosmosHub)   |
 |`chainRoutes`   |`string[]`  |true   |The list of IBC chains that this token needs to be forwarded through in order to be deposited into Carbon blockchain.    |1. You need to add **at least 1** blockchain network to this array.<br /><br />2. Blockchain networks in this array **MUST** match the valid chainName of the bridges listed under BridgeAll RPC call.<br /><br /> To view the values of BridgeAll RPC call, simply run `yarn get-bridges [network]` on the command line. Sample for mainnet: `yarn get-bridges mainnet`   |
 |`denomOnCarbon`   |`string`  |false   |Denom of token that is added to Carbon chain but still requires packet-forwarding (omit if this token is **NOT** added to Carbon chain)   |The denom in this field **MUST** match the token denoms listed under the Carbon [Tokens API](https://api.carbon.network/carbon/coin/v1/tokens?pagination.limit=10000).   |
+
+## DemexTradingLeagueConfig
+|Field   |Type   |Required   |Description   |Notes   |
+|---|---|---|---|---|
+|`promoMarkets`   |`string[]`   |true   |The array of market names which are designated as Promo markets. |The market names listed here **MUST** match the market names listed under the Carbon [Markets API](https://api.carbon.network/carbon/market/v1/markets?pagination.limit=10000). |
+|`currentMarketSymbol`   |`string`  |true   |The primary market symbol, mainly used as the market for the trading league page description/headers.  | The symbol in this field **MUST** match the token symbols listed under the Carbon [Tokens API](https://api.carbon.network/carbon/coin/v1/tokens?pagination.limit=10000).   |
```

Comment:
This is mainly for the reward token, maybe can do the following:
- `currentMarketSymbol` => `currentCompRewardSymbol`
- Rewrite the description to the following:
`The symbol of the reward token, mainly used to display the reward symbol and token on the Demex Trading League page`